File: programme/linear-search/linear-search.java
class LinearSearch
{
	public static void main (String[] args)
	{
  	int n = 6;
    int input[] = {10, 20, 80, 30, 60, 50};
    int target = 30;
    int result = 0;
    for(int i=0;i< n;i++){
        if(input[i] == target){
          System.out.print("Found at index "+ i);
          result = 1;
        }
    }
        if(result == 0){
          System.out.println("Target Not Found !!!");
     	  } 
  }
}

File: programme/linear-search/LinearSearch.cs
// Write a programme for linear search
using System;

public class Program
{
    public static void Main()
    {
        int[] input = { 10, 20, 80, 30, 60, 50 };
        int target = 30;
        int result = LinearSearch(input, target);
        if (result == -1)
            Console.WriteLine("Target Not Found");
        else
            Console.WriteLine("Target Found at Index : " + result);
    }

    public static int LinearSearch(int[] array, int target)
    {
        int n = array.Length - 1;
        for (int i = 0; i <= n; i++)
        {
            if (array[i] == target)
            {
                return i;
            }
        }

        return -1;
    }
}
